.travis.yml

```diff
@@ -21,9 +21,9 @@ addons:
 before_script:
 - |
   if [ $TRAVIS_OS_NAME = 'osx' ]; then
-    brew install python3 &&
-    virtualenv env -p python3 &&
+    virtualenv env &&
     source env/bin/activate &&
+    python --version &&
```

Comment:
I added this so that we can see in the logs which version of Python is being used. We could take it out, but I'm curious to see if it flips to Python 3 in the coming days. It's currently "Python 2.7.14".